jpgcoins/index.html

```diff
@@ -23,6 +23,7 @@ <h3> O Welligton Silva possui 2 jpg </h3>
 		<h3> O Daniel Ferraz possui 8 jpg </h3>
 		<h3> O Luan Ferreira possui 2 jpg </h3>
 		<h3> O Eduardo Cauli possui 10 jpg </h3>
+		<h3> O Henrique Grana possui 200 jpg </h3>
```

Comment:
**:warning: PLEASE DO NOT ATTEMPT TO VIOLATE THE SYSTEM. :warning:
:warning: YOU ARE NOW BLACK-LISTED :warning:** 

:black_circle: **CURRENT BLACKLIST:**
- [granaz](https://github.com/granaz)

:warning: **< This is an automatic response >** :warning: